cymetric/filters.py

```diff
@@ -175,20 +175,26 @@ def inventories(evaler, facilities=(), nucs=()):
     """
 
     # Get inventory table
-    df = evaler.eval('ExplicitInventory')
     agents = evaler.eval('AgentEntry')
 
     rdc_table = []  # because we want to get rid of the nuclide asap
     if len(nucs) != 0:
         nucs = format_nucs(nucs)
         rdc_table.append(['NucId', nucs])
 
+    conditionnal_agent = []
+    df = pd.DataFrame()
     if len(facilities) != 0:
         agents = agents[agents['Prototype'].isin(facilities)]
-        rdc_table.append(['AgentId', agents['AgentId'].tolist()])
+        dfs = []
+        for id in agents['AgentId'].tolist():
+            conditionnal_agent = ('AgentId','==',id)
+            dfs.append(evaler.eval('ExplicitInventory', conds=[conditionnal_agent]))
+            df = pd.concat(dfs)
```

Comment:
yes I did test it,
the thing is on small simulation, the time different is not noticeable , on big simulation I never reach that point as I was not able to load the full inventory table...

Is there a way to have `OR` condition in this ? I am looping over the conditions just because when I put multiple conditions it is an "AND"...